.maintain/chaostest/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "chaostest",
+  "description": "A cli for chaos testing on substrate",
+  "version": "0.0.0",
+  "author": "HarryHong",
+  "bin": {
+    "chaostest": "./bin/run"
+  },
+  "bugs": "https://github.com/HarryHong/chaostest/issues",
```

Comment:
I would guess it is easier to report these to `github.com/paritytech/substrate`.

Reply:
Bump.